libs/components/forms/src/lib/modules/radio/radio-group.component.ts

```diff
@@ -141,6 +143,12 @@ export class SkyRadioGroupComponent
     return this.#_tabIndex;
   }
 
+  /**
+   * The text to display as the radio group's label.
+   */
```

Comment:
elaborate that this is meant to be a replacement for `sky-radio-label`

Reply:
it isn't a replacement, it is a separate label